next/src/components/ui/TextLink/TextLink.tsx

```diff
@@ -0,0 +1,40 @@
+import Link from 'next/link';
+import styles from './TextLink.module.scss';
+import type { TextLinkTypes } from './TextLink.types';
+
+export default function TextLink({ children, ...props }: TextLinkTypes) {
+  return (
+    <Link href={props.href!} className={styles.link} {...props}>
```

Comment:
That's not a good idea to declare `props.href!`. Just do the explicit declaration of `href`.